spring-integration-cassandra/src/main/java/org/springframework/integration/cassandra/config/xml/CassandraParserUtils.java

```diff
@@ -91,7 +92,7 @@ public static void processOutboundTypeAttributes(Element element, ParserContext
 
 	}
 
-	public static boolean areMutuallyExclusive(String query, BeanDefinition statementExpressionDef,
+	public static boolean areMutuallyExclusive(String query, @Nullable BeanDefinition statementExpressionDef,
```

Comment:
And why this method is not `private` yet since you have already removed that redundant test?